pkgs/development/python-modules/ttfautohint-py/default.nix

```
@@ -23,10 +24,20 @@ buildPythonPackage rec {
     hash = "sha256-NTog461RpyHKo/Qpicj3tflehaKj9LlZEN9qeCMM6JQ=";
   };
 
-  postPatch = ''
-    substituteInPlace src/python/ttfautohint/__init__.py \
-      --replace-fail 'find_library("ttfautohint")' '"${lib.getLib ttfautohint}/lib/libttfautohint${stdenv.hostPlatform.extensions.sharedLibrary}"'
-  '';
+  postPatch =
+    [
+      ''
+        substituteInPlace src/python/ttfautohint/__init__.py \
+          --replace-fail 'find_library("ttfautohint")' '"${lib.getLib ttfautohint}/lib/libttfautohint${stdenv.hostPlatform.extensions.sharedLibrary}"'
+
+      ''
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [
+      ''
+        substituteInPlace src/python/ttfautohint/memory.py \
+          --replace-fail 'find_library("c")' '"${glibc}/lib/libc.so.6"'
+      ''
+    ];
 
   env.TTFAUTOHINTPY_BUNDLE_DLL = false;
```

```suggestion
  env.TTFAUTOHINTPY_BUNDLE_DLL = false;
```

If this is needed for a phase, it needs to be added like so:
```sh
buildPhase = ''
  preBuild
  env.TTFAUTOHINTPY_BUNDLE_DLL = false;
  # ... rest of build
  postBuild
];
```

---

I'm not the package maintainer, just patched the libc path, @jopejoe1 is the maintainer.

---

The `env` attrset is where environment variables used during the build process should go, which `TTFAUTOHINTPY_BUNDLE_DLL` is.